sdk/devcenter/developer-devcenter-rest/test/public/devBoxesTest.spec.ts

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { env, Recorder } from "@azure-tools/test-recorder";
+import { assert } from "chai";
+import { createRecordedClient, createRecorder } from "./utils/recordedClient";
+import { Context } from "mocha";
+import {
+  AzureDevCenterClient,
+  DevBoxesCreateDevBoxParameters,
+  getLongRunningPoller,
+  isUnexpected,
+} from "../../src/index";
+
+describe("DevCenter Dev Boxes Operations Test", () => {
+  let recorder: Recorder;
+  let client: AzureDevCenterClient;
+  let tenantId: string;
+  let devCenter: string;
+
+  beforeEach(async function (this: Context) {
+    recorder = await createRecorder(this);
+    tenantId = env["DEVCENTER_TENANT_ID"] || "<tenant id>";
+    devCenter = env["DEFAULT_DEVCENTER_NAME"] || "sdk-default-devcenter";
+    client = createRecordedClient(recorder, tenantId, devCenter, {
+      allowInsecureConnection: false,
+    });
+  });
+
+  afterEach(async function () {
+    await recorder.stop();
+  });
+
+  it("Create dev box", async function () {
+    // Build client and fetch required parameters
+    const projectName = env["DEFAULT_PROJECT_NAME"] || "sdk-default-project";
+    const poolName = env["DEFAULT_POOL_NAME"] || "sdk-default-pool";
+    const devBoxName = "SdkTestDevBox";
+    const userId = "me";
+
+    console.log(
+      `Running test for ${tenantId} -- ${devCenter} -- ${projectName} -- ${poolName} -- ${devBoxName}`
+    );
+
+    const devBoxCreateParameters: DevBoxesCreateDevBoxParameters = {
+      contentType: "application/json",
+      body: { poolName: poolName },
+    };
+
+    // Provision a dev box
+    const devBoxCreateResponse = await client
+      .path(
+        "/projects/{projectName}/users/{userId}/devboxes/{devBoxName}",
+        projectName,
+        userId,
+        devBoxName
+      )
+      .put(devBoxCreateParameters);
+
+    if (isUnexpected(devBoxCreateResponse)) {
+      console.log(`Unexpected ${JSON.stringify(devBoxCreateResponse)}`);
+      throw new Error(devBoxCreateResponse.body?.error.message);
+    }
+
+    assert.equal(devBoxCreateResponse.status, "201", "Dev Box creation should return 201 Created.");
+
```

Comment:
I notice that we may have format issue could you run format command to fix them?

```
>cd sdk/devcenter/developer-devcenter-rest
>rushx format
```

Reply:
Thanks for catching, all your help and feedback here is much appreciated! 😃